### PR TITLE
Implement batch action for editing the login state of users

### PR DESCRIPTION
**Pull request by JammingBen**
## Description
Adds a batch action to the users list in the admin settings which allows the admin to batch-edit the login state of users.

## Related Issue

- Fixes https://github.com/owncloud/web/issues/8798

## Screenshots
image

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>acceptance oCIS</strong> https://drone.owncloud.com/owncloud/web/34659/51/1
:boom: The oCISWebdavLockProtection tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oCIS</strong> https://drone.owncloud.com/owncloud/web/34659/50/1
:boom: The oCISBasic tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>e2e-tests oCIS</strong> https://drone.owncloud.com/owncloud/web/34659/12/1
:boom: The e2e-ocis tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>acceptance oC10</strong> https://drone.owncloud.com/owncloud/web/34659/14/1
:boom: The oC10FileFolderOperations tests pipeline failed. The build has been cancelled.

File: packages/web-app-admin-settings/src/composables/actions/users/useUserActionsEditLogin.ts [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+export const useUserActionsEditLogin = () => {
+  const { $gettext } = useGettext()
+
+  const actions = computed((): UserAction[] => [
+    {
+      name: 'edit-login',
+      icon: 'login-circle',
+      componentType: 'button',
+      class: 'oc-users-actions-edit-login-trigger',
+      label: () => $gettext('Edit Login'),
```
Comment (kulmann) [inline packages/web-app-admin-settings/src/composables/actions/users/useUserActionsEditLogin.ts:15]:
```suggestion
      label: () => $gettext('Edit login'),
```

**Review by kulmann — CHANGES_REQUESTED** [review]
The other modals have a lengthy, very detailed title and a short label. For text consistency, should this modal be changed as follows for consistency?

title: `Edit login for "Albert Einstein"` / `Edit login for 3 users`
dropdown label: `Login status`

**Pull request merged**